Source/Plugins/Core/com.equella.core/js/tsrc/settings/Search/facetedsearch/FacetedSearchSettingsModule.ts

```diff
@@ -21,14 +21,19 @@ import {
   BatchOperationResponse,
   groupErrorMessages,
 } from "../../../api/BatchOperationResponse";
+import { encodeQuery } from "../../../util/encodequery";
 
 export interface Facet {
   /**
-   * Name of a facet
+   * ID of a facet; being undefined means this facet is dirty.
```

Comment:
Might be good to have a quick line about the definition of dirty in this scenario.
```suggestion
   * ID of a facet; being undefined means this facet is dirty. (i.e., not saved to the server)
```